slidechain/postexport.go

```diff
@@ -0,0 +1,167 @@
+package slidechain
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"math"
+	"time"
+
+	"github.com/chain/txvm/crypto/ed25519"
+	i10rjson "github.com/chain/txvm/encoding/json"
+	"github.com/chain/txvm/errors"
+	"github.com/chain/txvm/protocol/bc"
+	"github.com/chain/txvm/protocol/txvm"
+	"github.com/chain/txvm/protocol/txvm/op"
+	"github.com/chain/txvm/protocol/txvm/txvmutil"
+	"github.com/stellar/go/xdr"
+)
+
+func (c *Custodian) doPostExport(ctx context.Context, assetXDR, anchor, txid []byte, amount, seqnum, peggedOut int64, exporter, temp string, pubkey []byte) error {
```

Comment:
"Post-export" is the wrong name for this function/this file/the other things here. This is _post-pegout_ logic.